src/dags/sport.py

```diff
@@ -0,0 +1,282 @@
+import operator, re, requests
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.dummy_operator import DummyOperator
+
+from http_fetch_operator import HttpFetchOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+
+
+from common import (
+    default_args,
+    pg_params,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    MessageOperator,
+)
+from postgres_check_operator import (
+    PostgresMultiCheckOperator,
+    COUNT_CHECK,
+    GEO_CHECK,
+)
+
+from sql.sport import ADD_GEOMETRY_COL, DEL_ROWS
+
+
+dag_id = "sport"
+variables = Variable.get(dag_id, deserialize_json=True)
+tmp_dir = f"/tmp/{dag_id}"
+files_to_download = variables["files_to_download"]
+data_endpoints = variables["data_endpoints"]
+files_to_SQL = {**files_to_download, **data_endpoints}
+total_checks = []
+count_checks = []
+geo_checks = []
+check_name = {}
+
+
+# needed to put quotes on elements in geotypes for SQL_CHECK_GEO
+def quote(instr):
+    return f"'{instr}'"
+
+# data connection to maps.amsterdam.nl
+def get_data(url, file_name):
+    """ calling the data endpoint on maps.amsterdam.nl """
+
+    # get data
+    data_url = url
+    data_data = requests.get(data_url)
+
+    # store data
+    with open(file_name, "w") as file:
+        file.write(data_data.text)
+    file.close()
+
+
+with DAG(
+    dag_id,
+    description="sportfaciliteiten, -objecten en -aanbieders",
+    default_args=default_args,
+    user_defined_filters=dict(quote=quote),
+) as dag:
+
+    # 1. Post message on slack
+    slack_at_start = MessageOperator(
+        task_id="slack_at_start",
+        http_conn_id="slack",
+        webhook_token=slack_webhook_token,
+        message=f"Starting {dag_id} ({DATAPUNT_ENVIRONMENT})",
+        username="admin",
+    )
+
+    # 2. create download temp directory to store the data
+    mk_tmp_dir = BashOperator(task_id="mk_tmp_dir", bash_command=f"mkdir -p {tmp_dir}")
+
+    # 3. download source 1: .csv or .geojson data from Objectstore
+    download_data_obs = [
+        HttpFetchOperator(
+            task_id=f"download_obs_{file_name}_{source_type}",
+            endpoint=f"{url}",
+            http_conn_id="api_data_amsterdam_conn_id",
+            tmp_file=f"{tmp_dir}/{file_name}.{re.sub('_.*', '', source_type)}",
+            output_type="text",
+        )
+        for source_type, data in files_to_download.items()
+        for file_name, url in data.items()
+    ]
+
+    # 4. Download source 2: .geojson from Maps.amsterdam.n
+    download_data_maps = [ 
+        PythonOperator(
```

Comment:
Was dit niet te doen met de HttpFetchOperator?

Reply:
Ah, ik weet niet meer precies waarom ik de httpfetch niet heb toegepast. Heb te veel dingen tegerlijkertijd lopen vrees ik, ik ga even kijken of het als nog lukt...

Reply:
Hi Jan, ik heb het werkend gekregen, Ziet er goed uit, nog puntjes op de i en dan doe ik een nieuwe commit